### PR TITLE
[Chore] Add Take survey integration test

### PR DESCRIPTION
**Pull request by sleepylee**
## What happened 👀

Add `Take Survey` integration test.
 
## Insight 📝

- Test the survey flow
- Test almost all the supported survey question type: star, emoji, heart, text, choices....and interact with those widgets.
- Complete the whole flow.
 
## Proof Of Work 📹

https://user-images.githubusercontent.com/13435717/134402024-1f28b112-9629-4d19-bb8b-2cb9142e4bd2.mov

### CODE REVIEW

File: integration_test/take_survey_widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import 'fake/fake_user_manager.dart';
+
+main() {
+  Get.testMode = true;
+
+  group('Validate take survey flow', () {
+    final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized()
+        as IntegrationTestWidgetsFlutterBinding;
+
+    binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
```
Comment (manh-t) [inline integration_test/take_survey_widget_test.dart:31]: does `framePolicy` make the test smoother? I feel like we are running a real app 🤪

Comment (sleepylee) [reply integration_test/take_survey_widget_test.dart:31]: I think so 😂

**Review by manh-t — APPROVED** [review]
🏃‍♂️ 👏

**Review by markgravity — APPROVED** [review]
💯 🛌

**Pull request merged**